classes/ActionScheduler_WPCLI_QueueRunner.php

```diff
@@ -0,0 +1,232 @@
+<?php
+
+/**
+ * WP CLI Queue runner.
+ *
+ * This class can only be called from within a WP CLI instance.
+ */
+class ActionScheduler_WPCLI_QueueRunner extends ActionScheduler_Abstract_QueueCleaner {
+
+	/** @var array */
+	protected $actions;
+
+	/** @var  ActionScheduler_ActionClaim */
+	protected $claim;
+
+	/** @var ActionScheduler_QueueCleaner */
+	protected $cleaner;
+
+	/** @var string */
+	protected $id;
+
+	/** @var ActionScheduler_FatalErrorMonitor */
+	protected $monitor;
+
+	/** @var \cli\progress\Bar */
+	protected $progress_bar;
+
+	/**
+	 * ActionScheduler_WPCLI_QueueRunner constructor.
+	 *
+	 * @param ActionScheduler_Store             $store
+	 * @param ActionScheduler_FatalErrorMonitor $monitor
+	 * @param ActionScheduler_QueueCleaner      $cleaner
+	 *
+	 * @throws Exception When this is not run within WP CLI
+	 */
+	public function __construct(
+		ActionScheduler_Store $store,
+		ActionScheduler_FatalErrorMonitor $monitor,
+		ActionScheduler_QueueCleaner $cleaner
+	) {
```

Comment:
I'd prefer to keep the function declarations consistent with all the others, which means all parameters declared on one line.

Reply:
I brought the arguments onto multiple lines since it was getting to be a bit long, but it's no big deal to put them back on one line.